pandas/plotting/_core.py

```diff
@@ -2051,8 +2051,16 @@ def plot_series(data, kind='line', ax=None,                    # Series unique
     Returns
     -------
     result :
```

Comment:
Slightly orthogonal to change but can you remove the trailing colon on the preceding line?

Reply:
yes, done!